core/build.gradle.kts

```diff
@@ -39,12 +39,16 @@ cargo {
     profile = findProperty("CARGO_PROFILE")?.toString() ?: currentFlavor
     extraCargoBuildArguments = listOf("--bin", libname!!)
     featureSpec.noDefaultBut(arrayOf(
-            "stream-cipher",
-            "aead-cipher-extra",
-            "logging",
-            "local-flow-stat",
-            "local-dns"))
+        "stream-cipher",
+        "aead-cipher-extra",
+        "logging",
+        "local-flow-stat",
+        "local-dns",
+        "security-iv-printable-prefix",
+        "aead-cipher-2022",
```

Comment:
Since you are using Rust nightly, consider adding `armv8` and `neon` for producing optimized code for ciphers.